components/automate-ui/src/app/entities/policy-files/policy-group.reducer.ts

```diff
@@ -0,0 +1,47 @@
+import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';
+import { set, pipe } from 'lodash/fp';
+import { EntityStatus } from 'app/entities/entities';
+import { PolicyFileActionTypes, PolicyFileActions } from './policy-file.action';
+import { PolicyFile } from './policy-file.model';
+
+export interface PolicyGroupEntityState extends EntityState<PolicyFile> {
+  getGroupsStatus: EntityStatus;
+  policyFile: {
```

Comment:
we can use only ``` policyfile: Policyfile[];``` No need to add another map in this

Reply:
Updated as ```policyfile: Policyfile[];```